public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -219,4 +220,20 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       return this.sourcesChartHelpText;
     }
   }
+  public expandRow(concepts, r) {
```

Comment:
Please use types

Reply:
Concept does not have expanded prop in model so using any.